index.html

```diff
@@ -151,19 +157,17 @@ <h3 id="noharm" data-export="" data-dfn-type="dfn">
   </h3>
   <p>
 When we are adding a feature or technology to the web,
-we will consider what harm it could do to society or groups,
+we will work to prevent or mitigate any harm it might cause society or groups,
 especially to vulnerable people.
+We consider a range of threat models that account for abuse scenarios
+at different scales, from societal to interpersonal.
 We will prioritize potential benefits for web users over potential benefits to web developers,
 content providers, user agents, advertisers or others in the ecosystem,
 in line with the
 <a data-cite="design-principles#priority-of-constituencies">priority of constituencies</a>.
-We will ensure the requirements and views of marginalized communities
-and underrepresented groups are heard and respected.
-We will build new web technologies in a collaborative manner
-according to open processes
-(for example, the <a href="https://www.w3.org/Consortium/Process/">W3C process</a>),
-and adhering to codes of conduct
-(such as the W3C <a href="https://www.w3.org/Consortium/cepc/">Code of Ethics and Professional Conduct</a>).
+We will actively seek to involve marginalized communities
+in the design of features and technologies,
+and ensure the views of underrepresented groups are heard and respected.
```

Comment:
I have a few minor quibbles with this:

1. "ensure" and "actively seek" are a bit at odds here.

I don't think that you can truly ensure that a system can allow all people similar access, underrepresented or otherwise. 

2. The point is to strive for due consideration (hearing, respecting, etc...) of the full diversity of people's interests (interests more so than views, but also views).

Maybe:

> We recognize the difficult in trying to ensure that our designs
> properly respect the interests and views of
> all of the people that might be affected by them.
> Despite the challenges,
> we commit to making the effort to learn and understand diverse perspectives,
> and will strive to reflect a respect for that diversity in the designs we produce.